src/CodeMockup/CodeMockup.stories.tsx

```diff
@@ -14,55 +14,57 @@ const meta: Meta = {
 
 export default meta
 
-const Template: Story<CodeMockupProps> = (args) => {
-  return <CodeMockup {...args} />
+export const Default: Story<CodeMockupProps> = (args) => {
+  return (<CodeMockup {...args} >
```

Comment:
For style consistency, can you add a newline inbetween the parenthesis, so this becomes:
```js
return (
     <CodeMockup>
     ...
     </CodeMockup>
)
```

Reply:
fixed it on [9a48907](https://github.com/daisyui/react-daisyui/pull/252/commits/9a489078979eacac01d9e6223870bd50ffb80fdf) 
not very sure this is what you meant tho